commands/rootCmd.go

```diff
@@ -0,0 +1,83 @@
+/*
+Package commands contains github-audit shell commands related logic.
+*/
+package commands
+
+import (
+	"context"
+	"os"
+
+	"github.com/maplelabs/github-audit/input"
+	"github.com/maplelabs/github-audit/internal/configurator"
+	"github.com/maplelabs/github-audit/internal/taskmanager"
+	"github.com/maplelabs/github-audit/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	//ConfigFile stores config.yaml location
+	ConfigFile string
+	log        logger.Logger
+)
+
+// rootCmd represents the base command which is executed in normal run of github-audit.
+var rootCmd = &cobra.Command{
+	Use:   "github-audit",
+	Short: "github-audit monitors github repositories",
+	Long: `github-audit monitors github commits , operations and sends to observability platform.
+To run github-audit use subcommand "start" 
```

Comment:
Can you add command for priting version?

Reply:
version command added